packager/README.md

```diff
@@ -0,0 +1,43 @@
+Prerequisites:
+
+Acquire the encrypted private key and TLS certificate (these are not in the
+repo), and place in `certs/privkey.pem.enc` and `certs/cert.pem.enc`
+respectively.
+
+Deploy to GAE:
+
+```sh
+# generate "dist" directory (contains valid AMP)
+$ npx gulp --gulpfile ../gulpfile.js build:packager
+
+# deploy
+gcloud app deploy --project=abe-packager
+```
+
+Run locally (needs Docker installed):
+
+```sh
+# generate "dist" directory (contains valid AMP)
+$ npx gulp --gulpfile ../gulpfile.js build:packager
+
+# build local Docker image called "amppkg"
+$ docker build -t amppkg .
+
+# create local container from Docker image, expose ports (uses CMD)
+$ docker run -p 8080:8080 --env PASSWORD=(pbpaste) amppkg
+
+# create (local) container from Docker image and provide shell (ignore CMD)
+$ docker run -it amppkg bash
+```
+
+Encrypt file:
+
+```sh
+openssl aes-256-cbc -md md5 -e -k $PASSWORD -in plain.pem -out encrypted.pem.enc
+```
+
+Decrypt file:
+
+```sh
+openssl aes-256-cbc -md md5 -d -k $PASSWORD -in encrypted.pem -out plain.pem.enc
```

Comment:
Should these be encpryted.pem.enc and plain.pem ? The ".enc" suffix seems flipped?

Reply:
They should! Thanks.